packages/synthetix-main/contracts/mixins/AccountRBACMixin.sol

```diff
@@ -6,41 +6,41 @@ import "../storage/AccountModuleStorage.sol";
 contract AccountRBACMixin is AccountModuleStorage {
     using SetUtil for SetUtil.Bytes32Set;
 
-    error RoleNotAuthorized(uint accountId, bytes32 role, address target);
-
-    bytes32 internal constant _ROLE_MODIFY = "ROLE_MODIFY";
-    bytes32 internal constant _ROLE_STAKE = "ROLE_STAKE";
-    bytes32 internal constant _ROLE_UNSTAKE = "ROLE_UNSTAKE";
-    bytes32 internal constant _ROLE_ASSIGN = "ROLE_ASSIGN";
-    bytes32 internal constant _ROLE_MINT = "ROLE_MINT";
-    bytes32 internal constant _ROLE_ADMIN = "ROLE_ADMIN";
-
-    modifier onlyRoleAuthorized(uint accountId, bytes32 role) {
-        if (!_authorized(accountId, role, msg.sender)) {
-            revert RoleNotAuthorized(accountId, role, msg.sender);
+    error PermissionDenied(uint accountId, bytes32 permission, address target);
+
+    bytes32 internal constant _MODIFY_PERMISSION = "MODIFY";
```

Comment:
Based on the talk with @noahlitvin , We need to remove _MODIFY_PERMISSION and use _ADMIN_PERMISSION instead.